Makefile

```diff
@@ -129,10 +130,14 @@ apply: $(KO) $(KODATA) ## Deploy the controller into the current kubernetes clus
 delete:  ## Delete controller from current kubernetes cluster.
 	helm uninstall dev --namespace ${CLUSTER_NAMESPACE}
 
+.PHONY: ecr-login
+ecr-login: ## Login to default AWS ECR repository.
```

Comment:
This target doesn't appear to be used anywhere. Is it worth keeping (e.g., for testing, etc.)?

Reply:
It's just meant as a convenience.  The dev setup guide was updated to use the new target but I missed adding it to this commit.  Nice catch 👍